app/survey_config/survey_config.py

```diff
@@ -29,7 +29,7 @@ class SurveyConfig:
     title_logo_alt: Optional[str] = None
     accessibility_url: str = f"{ONS_URL}/help/accessibility/"
     what_we_do_url: str = f"{ONS_URL}/aboutus/whatwedo/"
-    header_logo: Optional[str] = None
+    header_logo: Optional[bool] = False
```

Comment:
Since the default is `False` (bool) this should just now be `bool`, instead of `Optional[bool]`